semgrep-core/src/engine/Match_tainting_mode.mli

```diff
@@ -20,6 +20,14 @@ val taint_config_of_rule :
   unit) ->
   Dataflow_tainting.config * debug_taint * Matching_explanation.t list
 
+val check_fundef :
```

Comment:
Maybe a small comment explaining the function and its arguments.
We should try to have better documentation in our .mli.

Reply:
True, thanks for calling me out here.